content/pages/sc24.html

```diff
@@ -0,0 +1,70 @@
+<html>
+<head><title>sc24</title></head>
+<body>
+<section class="white_bg">
+    <div class="container">
+        <div class="space_half"></div>
+        <div class="row" >
+            <div class="col-lg-12 text-center">
+                <h4 class="green_font"><i class="green_font fa fa-book"></i> Supercomputing 2024</h4>
+            </div>
+        </div>
+        <hr class="small_centered_line_green">
+        <div class="col-lg-offset-1 col-lg-10 col-md-offset-2 col-md-8 col-sm-offset-1 col-sm-10 col-xs-offset-0 col-xs-12 text-left" >
+            <div class="col-xs-12 text-center"><img src="{static}/images/irods_sc24_logo.png" style="width:100%;"></div>
+        </div>
+        <hr class="col-xs-offset-2 col-xs-8"/>
+<!--
+            <a href="{static}/images/sc24_floor_plan.pdf"><img src="{static}/images/presentation_map_button.jpg" class="member_logo"></a>
+            <a href="{static}/images/sc24_iRODS_schedule.png"><img src="{static}/images/presentation_sched_button.jpg" class="member_logo"></a>
+-->
+        <hr class="col-xs-offset-2 col-xs-8"/>
+        <div class="col-lg-offset-1 col-lg-10 col-md-offset-2 col-md-8 col-sm-offset-1 col-sm-10 col-xs-offset-0 col-xs-12 text-center" >
+            We will be holding a FREE mini-workshop at SC24 in Atlanta, Georgia:<br><br>
+            <h3>iRODS Updates and Troubleshooting</h3><br>
+
+            <p>
+            The open source iRODS (Integrated Rule-Oriented Data System) data management platform presents a
+            virtual filesystem, metadata catalog, and policy engine designed to give organizations maximum
+            control and flexibility over their data management practices and enforcement.
+            </p>
+
+            <p>
+            This workshop will share a year's worth of updates including work done in the server, a new PAM 
+            Interactive Authentication Plugin, HTTP and OpenID Connect functionality, the S3 API, and 3 new client
+            libraries written by our summer interns.  We'll also talk about the future of iRODS 5.0 and what it
+            may mean for integrations to come.
+            </p>
+
+            <p>
+            In addition, join us for troubleshooting time with the experts.  Bring us your tricky questions and
+            weird scenarios.  Come learn from others.
+            </p>
+
+            <p>
+            The iRODS Consortium was started as an open-source software development organization in 2013 by
```

Comment:
`open-source` on this line has a hyphen while the one on L27 does not.

Reply:
ah, yes, removed the hyphen.